aya-bpf-macros/src/expand.rs

```diff
@@ -200,20 +200,37 @@ impl SkMsg {
 pub struct Xdp {
     item: ItemFn,
     name: Option<String>,
+    multibuffer: bool,
 }
 
 impl Xdp {
     pub fn from_syn(mut args: Args, item: ItemFn) -> Result<Xdp> {
-        let name = name_arg(&mut args)?;
-
-        Ok(Xdp { item, name })
+        let name = pop_arg(&mut args, "name");
+        let mut multibuffer = false;
+        if let Some(s) = pop_arg(&mut args, "multibuffer") {
```

Comment:
I agree with the other comment that this should be called "frags"

Reply:
Renamed